TOOLS/dylib-unhell.py

```diff
@@ -94,11 +84,11 @@ def process_libraries(libs_dict, binary, processed = []):
         if src in libs_dict[binary]:
             install_name_tool_change(src, name, binary)
 
-        for p in processed:
+        for p in libs_set:
             if p in libs_dict[src]:
                 install_name_tool_change(p, lib_name(p), dst)
 
-    return ls
+    return
```

Comment:
this can be removed

Reply:
Removed in 78161279713c9ea766967545ba482cc5017083b2.